src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -45,6 +45,7 @@ public SortCommand(Comparator<Person> comparator) {
     public CommandResult execute(Model model, CommandHistory history) {
         requireNonNull(model);
         model.updateSortedPersonList(comparator);
+        model.commitAddressBook();
```

Comment:
if the `SortCommand` is committed to `VersionedAddressBook` should the `FindCommand` be as well?